### PR TITLE
First Weely Project of Microverse Full-Stack Program

### PR DESCRIPTION
**Pull request by umair-rao**
In this PR I have files of html and css as well as correcting using linters.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,27 @@
...
+
+Prerequisites
+Setup
+Install: VS Code, Chorme Browser
+Authors
+👤 Author1
+
+GitHub: @umair-rao
+LinkedIn: https://www.linkedin.com/in/engr-umair-muqeem/
+
```
Comment (nedjwab) [inline README.md:7]: Great Updating your README file 👀 , But kindly follow the [Template](https://github.com/microverseinc/readme-template), so it looks more professional.🤓

**Review by nedjwab — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @umair-rao 

Nice job, you are almost there.



## (Highlights) Good Points: :
- Good implementation of some of the project requirements  ✔
- GitHub flow was correctly used ✔.

## (Changes Required) Aspects to improve: ♻️
You did great implementing some of the required changes, but you still have some issues: 

[ ] Kindly add a description to your PR ( example: Adding Header .... )🤓.

[ ] kindly add a summary to your PR (example: In This PR i ...) 🤓.

[ ] kindly Set up the linters for your repository (add the stylelint, website, and Lighthouse configuration. you can find the steps[ here](https://github.com/microverseinc/linters-config/tree/master/html-css).
 
## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md (hunk 1 above)
Comment (umair-rao) [reply README.md:7]: File updated according to template.

**Comment by umair-rao** [comment]
All advised changes have been made. Review is requested.

**Review by franclobo — APPROVED** [review]
Hi @umair-rao, 

# STATUS: APPROVED 😃 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- No linter errors ✔️ 
- Descriptive commits ✔️ 
- Correct GitHub flow ✔️ 

## Optional suggestions
- [x] **[OPTIONAL]** Please 🙏🏽 add your name in the author's section for a professional README file.

image

- [x] **[OPTIONAL]** Kindly make sure that each section looks like the README template and that the link work as expected. Please 🙏🏽 add the LICENSE file to your repository.
image



_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**